libsplinter/src/peer/error.rs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Enum for errors that can be raised by the PeerManager.
```

Comment:
Should this be plural “Enums” - perhaps we should use the generic term “Types”, as this module could contain struct errors as well